GCRCatalogs/cosmodc2.py

```diff
@@ -430,7 +432,7 @@ def _generate_quantity_modifiers(self):
             'magnification': (lambda mag: np.where(mag < 0.2, 1.0, mag), 'magnification'),
             'halo_id':       'uniqueHaloID',
             'halo_mass':     (lambda x: x/self.cosmology.h, 'hostHaloMass'),
-            'is_central':    (lambda x: x.astype(bool), 'isCentral'),
+            'is_central':    (lambda x: x.astype(np.bool), 'isCentral'),
```

Comment:
`np.bool` has been deprecated
```suggestion
            'is_central':    (lambda x: x.astype(bool), 'isCentral'),
```